PyInstaller/hooks/hook-enchant.py

```diff
@@ -18,3 +20,24 @@
     for d in files:
         for f in d[1]:
             datas.append((f, d[0]))
+elif is_darwin:
+    # Collecting dictionaries only works when pyenchant is installed via pip since the dictionaries installed via pip
```

Comment:
Do you have any suggestion how this could be improved to collect dictionaries that were installed via macports or homebrew?